### PR TITLE
chore: Fix PR comments

### PR DESCRIPTION
**Pull request by dr1rrb**
Fix not addressed comments of https://github.com/unoplatform/uno/pull/8406

### CODE REVIEW

File: src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 namespace System.Runtime.CompilerServices;
 
-#if !NET5_0 && !NET6_0
+#if !NET5_0 && !NET6_0_OR_GREATER
```
Comment (Youssef1313) [inline src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs:6]:
```suggestion
#if !NET5_0_OR_GREATER
```

File: src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
 #pragma warning disable CS0436
```
Comment (Youssef1313) [inline src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs:7]:
```suggestion
```

File: src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs [hunk 3]
```diff
@@ -28,7 +28,7 @@ partial class ListViewBase
 		public event DragItemsStartingEventHandler DragItemsStarting;
 		public event TypedEventHandler<ListViewBase, DragItemsCompletedEventArgs> DragItemsCompleted;
 
-		protected bool _isProcessingReorder;
+		private bool _isProcessingReorder;
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs:31]: Unrelated to your changes. I see that this field is only accessed on `ListView` due to:

https://github.com/unoplatform/uno/blob/685f9c31d00b9adee8d6cb43aeebb2b4601c85e9/src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs#L238

Is this specific only to `ListView? Is it not applicable to `GridView`?

File: src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs (hunk 1 above)
Comment (dr1rrb) [reply src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs:6]: `NET5_0_OR_GREATER` doesn't exist ^^

Comment (Youssef1313) [reply src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs:6]: @dr1rrb I thought it should exist 👀 

At least it does in another project, but what would make it available in one but not the other?

https://github.com/unoplatform/uno/blob/896018279fd913fc88ae1e5eb10cb3e1b6f7f1c3/src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.cs#L89

File: src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs (hunk 3 above)
Comment (dr1rrb) [reply src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs:31]: D'oh ... you're right ! Going to check for `ListViewBase` instead ^^

File: src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs (hunk 2 above)
Comment (dr1rrb) [reply src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs:7]: Indeed, I copy the file coming from uno.extensions, but this is actually not needed

**Review by carldebilly — APPROVED** [review]

File: src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs (hunk 1 above)
Comment (dr1rrb) [reply src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs:6]: Well ... I only trusted what said @jeromelaban 🤪

**Review by jeromelaban — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

**Comment by jeromelaban** [comment]
@dr1rrb the build fails because of 
```
src\Uno.UI\UI\Xaml\Controls\ListViewBase\ListViewBase.DragDrop.cs(31,16): Error CS0414: The field 'ListViewBase._isProcessingReorder' is assigned but its value is never used
```

**Pull request merged**